### ISSUE TITLE
Mistaken commit to buildinfo.json

### ISSUE THREAD

**Issue by bplein** [factoriotools/factorio-docker#478]
I didn't realize i had comitter privileges here and thought I was committing a change to buildinfo.json on my fork. 

We need to roll this back.

**Comment by Fank** [comment]
@bplein did you resolved it already?

**Comment by bplein** [comment]
Yeah, I just erroneously updated this repo vs my fork. 

There was a hiccup in the upstream update from Factorio.  They shipped a bad version of 1.1.98 and then updated it again and I was trying to force a build of my fork. 

I don’t need committer status here. I can submit PRs if I ever get involved again. Keep this idiot (me) away from the keyboard. 

Btw I did some updates on a fork of the watchdog. Basically if you restart it, it tries too many branches and starts hitting CDN limits. I put in an env variable that represents the lowest version to download and check. If you set it to 1.0.0 it ignores all the older versions. 

I also started mounting as a volume where it stores its copy of the SHA256 sums. This seems to be working swimmingly. And I don’t even know golang. 😀
